Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:2.5
+
+RUN gem install bundler jekyll
+
+RUN mkdir /srv/jekyll
+
+WORKDIR /srv/jekyll
```

Comment:
You can just use `WORKDIR path`, if `path` does not exist, then it will be created.
Why do we need `/srv/jekyll`? can't we just use `/jekyll` or `/site`?

Reply:
We don’t need `/srv/jekyll` specifically, it’s the path used by the official `jekyll` image, so I’ve used the same.